src/xmlsec/__init__.py

```diff
@@ -313,6 +314,27 @@ def _verify(t, keyspec, sig_path=".//{%s}Signature" % NS['ds'], drop_signature=F
             this_cert = xmlsec.crypto.from_keyspec(keyspec, signature_element=sig)
             log.debug("key size: {!s} bits".format(this_cert.keysize))
 
+            # Try verification by CA signed signing certificate
```

Comment:
This seems to turn on verification by default as soon as there is a signing certificate in metadata. Is this really what we want?

Reply:
I discussed this with my collegues at SURF and indeed they indicated anyone that has CA=true in BasicConstraints know what she's doing and presumably wanted to CA sign the signing certificate. And the following tests (verify_directly_issued_by) excludes any misconceptions about that?

Reply:
And no, it's not about the certificate in metadata, only the one supplied as verification cert, as far as I could understand the code paths?

Reply:
I guess that argument makes sense

Reply:
I would very much like to have a couple of testcases though: one where we test the happy path and one were we stick ca=true in an end-entity certificate but not trigger pkix validation

Reply:
From some local testing I can at least confirm we will _not_ trigger the CA code path if the end-entity certificate has CA=true (validation disabled) and still have a succesful pyff run.